client/src/components/DataFiles/tests/DataFiles.test.js

```diff
@@ -44,7 +44,7 @@ describe('DataFiles', () => {
       history
     )
     expect(history.location.pathname).toEqual(
-      '/workbench/data/tapis/private/frontera.home.username/'
+      '/workbench/data/tapis/private/corral.home.username/'
```

Comment:
Should this be `frontera.home.username` since the systems fixture has it that way?

Reply:
I added more systems to the fixture, and since the Data Files by default loads the first item in the systems array, `corral.home.username` is correct